crypto/modes/asm/ghashv8-armx.pl

```diff
@@ -18,11 +18,9 @@
 #
 # June 2014
 #
-# Initial version was developed in tight cooperation with Ard
-# Biesheuvel <ard.biesheuvel@linaro.org> from bits-n-pieces from
-# other assembly modules. Just like aesv8-armx.pl this module
-# supports both AArch32 and AArch64 execution modes.
-#
+# Initial version was developed in tight cooperation with Ard Biesheuvel
+# of Linaro from bits-n-pieces from other assembly modules. Just like
+# aesv8-armx.pl this module supports both AArch32 and AArch64 execution modes.
```

Comment:
I have to insist on empty line after this paragraph. Without it next line can be viewed as time stamp for this paragraph, while it's not.

Reply:
I moved the blank line before the July timestamp.